src/test/java/org/sagebionetworks/bridge/validators/AssessmentValidatorTest.java

```diff
@@ -132,4 +138,17 @@ public void ownerIdEmpty() {
         assessment.setOwnerId("\t");
         assertValidatorMessage(validator, assessment, "ownerId", CANNOT_BE_BLANK);
     }
+    @Test
+    public void propertyInfoInvalid() {
```

Comment:
Test case with a valid Property Info?

Reply:
The assessment created for validAssessment contains all fields filled out and valid (including customizatonFields).